### PR TITLE
Remove use_wsio cmake flag

### PR DESCRIPTION
**Pull request by dcristoloveanu**
Remove use_wsio cmake flag

### CODE REVIEW

File: build_all/linux/build.sh [hunk 1]
```diff
@@ -7,7 +7,6 @@ set -e
 script_dir=$(cd "$(dirname "$0")" && pwd)
 build_root=$(cd "${script_dir}/../.." && pwd)
 run_unit_tests=ON
-use_wsio=OFF
 run_valgrind=0
 build_folder=$build_root"/cmake/uamqp_linux"
 run_unittests=OFF
```
Comment (ewertons) [inline build_all/linux/build.sh]: Gotta remove this too:

build_all\linux\build.sh:21:    echo " --use-websockets              Enables the support for AMQP over WebSockets."

File: CMakeLists.txt [hunk 2]
```diff
@@ -44,7 +44,6 @@ option(run_unittests "set run_unittests to ON to run unittests (default is OFF)"
 option(skip_samples "set skip_samples to ON to skip building samples (default is OFF)[if possible, they are always built]" OFF)
```
Comment (ewertons) [inline CMakeLists.txt]: One more to remove:

jenkins\windows_c.cmd:13:call build.cmd --use-websockets %*

File: build_all/windows/build.cmd [hunk 3]
```diff
@@ -68,10 +66,6 @@ if %build-platform% == x64 (
 )
 goto args-continue
 
-:arg-use-websockets
-set CMAKE_use_wsio=ON
-goto args-continue
-
 :arg-build-nuget
 shift
 if "%1" equ "" call :usage && exit /b 1
```
Comment (ewertons) [inline build_all/windows/build.cmd]: These must be removed too:

build_all\windows\build.cmd:115:    cmake %build-root% -Duse_wsio:BOOL=%CMAKE_use_wsio%
build_all\windows\build.cmd:125:    cmake %build-root% -Duse_wsio:BOOL=%CMAKE_use_wsio% -G "Visual Studio 14 Win64"
build_all\windows\build.cmd:135:    cmake %build-root% -Duse_wsio:BOOL=%CMAKE_use_wsio% -G "Visual Studio 14 ARM"
build_all\windows\build.cmd:140:    cmake %build-root% -Drun_unittests:BOOL=ON -Duse_wsio:BOOL=%CMAKE_use_wsio%
build_all\windows\build.cmd:144:    cmake %build-root% -Drun_unittests:BOOL=ON -Duse_wsio:BOOL=%CMAKE_use_wsio% -G "Visual Studio 14 ARM"
build_all\windows\build.cmd:148:    cmake %build-root% -Drun_unittests:BOOL=ON -Duse_wsio:BOOL=%CMAKE_use_wsio% -G "Visual Studio 14 Win64"

Comment (dcristoloveanu) [inline build_all/windows/build.cmd]: Interesting how I missed this. Removed.

---
In reply to: [107512198](https://github.com/Azure/azure-uamqp-c/pull/58#discussion_r107512198) [](ancestors = 107512198)

File: build_all/linux/build.sh (hunk 1 above)
Comment (dcristoloveanu) [inline build_all/linux/build.sh]: Removed.

---
In reply to: [107511123](https://github.com/Azure/azure-uamqp-c/pull/58#discussion_r107511123) [](ancestors = 107511123)

File: CMakeLists.txt (hunk 2 above)
Comment (dcristoloveanu) [inline CMakeLists.txt]: Removed.

---
In reply to: [107511461](https://github.com/Azure/azure-uamqp-c/pull/58#discussion_r107511461) [](ancestors = 107511461)

**Pull request merged**